rest/server_context.go

```diff
@@ -538,13 +538,23 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		channelIndexOptions = nil
 	}
 
+	var revCacheOptions db.RevisionCacheOptions
+
 	var revCacheSize uint32
 	if config.RevCacheSize != nil && *config.RevCacheSize > 0 {
 		revCacheSize = *config.RevCacheSize
 	} else {
 		revCacheSize = db.KDefaultRevisionCacheCapacity
 	}
 
+	var revCacheShardNum uint16
+	revCacheShardNum = 0
```

Comment:
I guess this will be updated as part of the external/config options PR?

Reply:
Yes. Currently set to 0 to preserve default value and then will match this to the appropriate config option as part of CBG-355